share/ocr-languages.json

```diff
@@ -1,66 +1,130 @@
     "Belarusian": "bel",
     "Bengali": "ben",
+    "Tibetan Standard": "bod",
+    "Bosnian": "bos",
+    "Breton": "bre",
     "Bulgarian": "bul",
     "Catalan": "cat",
+    "Cebuano": "ceb",
     "Czech": "ces",
     "Chinese - Simplified": "chi_sim",
+    "Chinese - Simplified (vertical)": "chi_sim_vert",
     "Chinese - Traditional": "chi_tra",
+    "Chinese - Traditional (vertical)": "chi_tra_vert",
     "Cherokee": "chr",
+    "Corsican": "cos",
+    "Welsh": "cym",
     "Danish": "dan",
+    "Danish - Fraktur": "dan_frak",
     "German": "deu",
+    "German - Fraktur": "deu_frak",
+    "Divehi": "div",
+    "Dzongkha": "dzo",
     "Greek": "ell",
     "English": "eng",
     "English, Middle (1100-1500)": "enm",
     "Esperanto": "epo",
+    "Math / equation detection module": "equ",
     "Estonian": "est",
     "Basque": "eus",
+    "Faroese": "fao",
+    "Persian": "fas",
+    "Filipino": "fil",
     "Finnish": "fin",
     "French": "fra",
     "Frankish": "frk",
     "French, Middle (ca.1400-1600)": "frm",
+    "Frisian (Western)": "fry",
+    "Gaelic (Scots)": "gla",
+    "Irish": "gle",
     "Galician": "glg",
-    "Greek, Ancient, to 1453": "grc",
+    "Greek, Ancient (to 1453)": "grc",
+    "Gujarati": "guj",
+    "Hatian": "hat",
     "Hebrew": "heb",
     "Hindi": "hin",
     "Croatian": "hrv",
     "Hungarian": "hun",
+    "Armenian": "hye",
+    "Inuktitut": "iku",
     "Indonesian": "ind",
     "Icelandic": "isl",
     "Italian": "ita",
     "Italian - Old": "ita_old",
+    "Javanese": "jav",
     "Japanese": "jpn",
+    "Japanese (vertical)": "jpn_vert",
     "Kannada": "kan",
     "Georgian": "kat",
+    "Old Georgian": "kat_old",
+    "Kazakh": "kaz",
+    "Khmer": "khm",
+    "Kyrgyz": "kir",
+    "Kurmanji (Kurdish - Latin Script)": "kmr",
     "Korean": "kor",
+    "Korean (vertical)": "kor_vert",
+    "Lao": "lao",
+    "Latin": "lat",
     "Latvian": "lav",
     "Lithuanian": "lit",
+    "Luxembourgish": "ltz",
     "Malayalam": "mal",
+    "Marathi": "mar",
     "Macedonian": "mkd",
     "Maltese": "mlt",
+    "Mongolian": "mon",
+    "Maori": "mri",
     "Malay": "msa",
+    "Burmese": "mya",
+    "Nepali": "nep",
     "Dutch": "nld",
     "Norwegian": "nor",
+    "Occitan (post 1500)": "oci",
+    "Oriya": "ori",
+    "Punjabi": "pan",
     "Polish": "pol",
     "Portuguese": "por",
+    "Pashto": "pus",
+    "Quechua": "que",
     "Romanian": "ron",
     "Russian": "rus",
+    "Sanskrit": "san",
+    "Sinhala": "sin",
     "Slovakian": "slk",
+    "Slovak - Fraktur": "slk_frak",
+    "Slovenian": "slv",
+    "Sindhi": "snd",
     "Spanish": "spa",
     "Spanish; Castilian - Old": "spa_old",
     "Albanian": "sqi",
     "Serbian": "srp",
+    "Serbian (Latin)": "srp_latn",
+    "Sundanese": "sun",
     "Swahili": "swa",
     "Swedish": "swe",
+    "Syriac": "syr",
     "Tamil": "tam",
+    "Tatar": "tat",
     "Telugu": "tel",
-    "Filipino": "tgl",
+    "Tajik": "tgk",
+    "Tagalog (new - Filipino)": "tgl",
```

Comment:
The change from `Filipino` to `Tagalog (new - Filipino)` is probably going to give us some issues since we were storing in the settings the keys (`Filipino`) and not the values (`tgl`). And the tesseract developers seem to consider the keys as OK to change, but they try to preserve the values.

In the following code, `ocr_languages["Filipino"]` will no longer exist and it'll probably raise an IndexError:

https://github.com/freedomofpress/dangerzone/blob/09a0e51c3f0f1b9dccf4e41cc3983e17207afda0/dangerzone/gui/main_window.py#L725-L728

Reply:
Yes, I also recall that discussion. Maybe somewhere else it catches that exception.

Reply:
Actually, we'll never get to an exception, because this is what happens before this code:

https://github.com/freedomofpress/dangerzone/blob/56c5d77afde00b7ee0b988548036b4056aaaa5b2/dangerzone/gui/main_window.py#L526-L528

https://github.com/freedomofpress/dangerzone/blob/56c5d77afde00b7ee0b988548036b4056aaaa5b2/dangerzone/gui/main_window.py#L649

So, invalid languages would lead to replacing them with "Afrikaans" in the settings.

Reply:
You're right. And sorry for making you dig into this again.